app/src/main/java/com/cdio/solitaire/controller/GameStateController.kt

```diff
@@ -116,15 +120,18 @@ class GameStateController {
                 move.targetStack!!
             )
             MoveType.MOVE_FROM_FOUNDATION,
-            MoveType.MOVE_FROM_TALON,
+            MoveType.MOVE_FROM_TALON -> moveCard(move.sourceStack!!, move.targetStack!!)
             MoveType.MOVE_TO_FOUNDATION -> cardToUpdate =
-                moveCard(move.sourceStack!!, move.targetStack!!)
+                moveToFoundation(move.sourceStack!!, move.sourceCard!!)
             MoveType.FLIP_TALON -> flipTalon()
             MoveType.DRAW_STOCK -> {
                 drawFromStock()
                 cardToUpdate = cardToUpdate(gameState.talon)
             }
             MoveType.DEAL_CARDS -> throw Exception("DEAL_CARDS is not for use here.")
+            else -> {
+            // println(move.moveType)
+            }
```

Comment:
Since print statement is commented out, the whole `else` block should also be commented out, so we don't have an empty block.